eng/Versions.props

```diff
@@ -255,5 +255,26 @@
     <!-- installer version, for testing workloads -->
     <MicrosoftDotnetSdkInternalVersion>9.0.100-preview.4.24175.5</MicrosoftDotnetSdkInternalVersion>
     <SdkVersionForWorkloadTesting>$(MicrosoftDotnetSdkInternalVersion)</SdkVersionForWorkloadTesting>
+    <MicrosoftNETRuntimeEmscripten3134Nodeosxarm64Version>9.0.0-preview.4.24175.6</MicrosoftNETRuntimeEmscripten3134Nodeosxarm64Version>
```

Comment:
Could we use a common variable for all of these duplicates? It would make it stand out more visually when one of them has to be a different version for some reason